### PR TITLE
Add poll_interval

### LINKED ISSUE #2473: Deferrable operator should also have poll_interval
We added poll_interval to our armadaOperator.  https://github.com/armadaproject/armada/pull/2444

I noticed that we didn't add this API field to the deferrable operator.  We should add poll_interval as a parameter (https://github.com/armadaproject/armada/blob/master/third_party/airflow/armada/operators/armada_deferrable.py#L83)

┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/BATCH-286) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by sarthaksarthak9**
Fixes #2473 

#### Special notes for your reviewer:
Thank you for reviewing

┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/BATCH-591) by [Unito](https://www.unito.io)

### CODE REVIEW

File: docs/python_airflow_operator.md [hunk 1]
```diff
@@ -81,6 +81,22 @@ Runs an Armada job and calls the job_service_client for polling.
```
Comment (kannon92) [inline docs/python_airflow_operator.md:83]: These docs don’t match the code you have here.

Comment (sarthaksarthak9) [reply docs/python_airflow_operator.md:83]: yah I am updating them!!

**Comment by sarthaksarthak9** [comment]
@kannon92 I did changes and also run tox -e docs-check
 (armada38) sarthak_negi@:~/armada/third_party/airflow$ python3 -m tox -e docs-check
.pkg: _optional_hooks> python /home/sarthak_negi/armada/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
.pkg: get_requires_for_build_sdist> python /home/sarthak_negi/armada/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
.pkg: build_sdist> python /home/sarthak_negi/armada/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
docs-check: install_package_deps> python -I -m pip install 'apache-airflow>=2.6.3' armada-client 'grpcio-tools>=1.46.3' 'grpcio>=1.46.3' sphinx-jekyll-builder==0.3.0 sphinx-markdown-builder==0.5.5 sphinx-toolbox==3.2.0b1 sphinx==7.0.1 'types-protobuf>=3.19.22'
docs-check: install_package> python -I -m pip install --force-reinstall --no-deps /home/sarthak_negi/armada/third_party/airflow/.tox/.tmp/package/3/armada_airflow-0.5.1.tar.gz
docs-check: commands[0] /home/sarthak_negi/armada/third_party/airflow/docs> sphinx-build -M jekyll source build
Running Sphinx v7.0.1
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
writing output...
building [jekyll]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
reading sources...
looking for now-outdated files... none found
no targets are out of date.
build succeeded.

The jekyll files are in build/jekyll.
docs-check: commands[1] /home/sarthak_negi/armada/third_party/airflow/docs> bash -c 'grep -v '"'"'^date:'"'"' build/jekyll/python_airflow_operator.md > /tmp/python_airflow_operator_generated.md'
docs-check: commands[2] /home/sarthak_negi/armada/third_party/airflow/docs> diff /tmp/python_airflow_operator_generated.md ../../../docs/python_airflow_operator.md
.pkg: _exit> python /home/sarthak_negi/armada/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
  docs-check: OK (62.94=setup[62.42]+cmd[0.48,0.04,0.00] seconds)
  congratulations :) (63.20 seconds)

Still pr is falling could you pls give me a lead

**Comment by kannon92** [comment]
yea it seems like something went wrong here.  https://github.com/armadaproject/armada/pull/2808

I'll aim to fix it.

**Comment by sarthaksarthak9** [comment]
> yea it seems like something went wrong here. #2808
> 
> I'll aim to fix it.

So, this is the same reason for my prev PR also?

**Comment by kannon92** [comment]
> @kannon92 pls review the PR if the issue is fixed!! [#2808](https://github.com/armadaproject/armada/pull/2808)

I fixed that issue but it looks like docs are wrong.  

Please run `tox -e docs` and push those changes.

**Comment by sarthaksarthak9** [comment]
> > @kannon92 pls review the PR if the issue is fixed!! [#2808](https://github.com/armadaproject/armada/pull/2808)
> 
> I fixed that issue but it looks like docs are wrong.
> 
> Please run `tox -e docs` and push those changes.

ok I will do it once again

**Review by kannon92 — APPROVED** [review]
Once this is green I think its mergeable!  Thank you!

**Comment by sarthaksarthak9** [comment]
> Once this is green I think its mergeable! Thank you!

You always guided me , your welcome

**Comment by sarthaksarthak9** [comment]
@Sharpz7  could you pls review the changes !!

**Comment by sarthaksarthak9** [comment]
@kannon92  I got a success message (armada38) sarthak_negi@LAPTOP-68O9S7GK:~/armada/third_party/airflow$ python3 -m tox -e docs-check
.pkg: _optional_hooks> python /home/sarthak_negi/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
.pkg: get_requires_for_build_sdist> python /home/sarthak_negi/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
.pkg: build_sdist> python /home/sarthak_negi/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
docs-check: install_package_deps> python -I -m pip install 'apache-airflow>=2.6.3' armada-client 'grpcio-tools>=1.46.3' 'grpcio>=1.46.3' sphinx-jekyll-builder==0.3.0 sphinx-markdown-builder==0.5.5 sphinx-toolbox==3.2.0b1 sphinx==7.1.2 'types-protobuf>=3.19.22'
docs-check: install_package> python -I -m pip install --force-reinstall --no-deps /home/sarthak_negi/armada/third_party/airflow/.tox/.tmp/package/3/armada_airflow-0.5.3.tar.gz
docs-check: commands[0] /home/sarthak_negi/armada/third_party/airflow/docs> sphinx-build -M jekyll source build
Running Sphinx v7.1.2
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
writing output...
building [jekyll]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
reading sources...
looking for now-outdated files... none found
no targets are out of date.
build succeeded.

The jekyll files are in build/jekyll.
docs-check: commands[1] /home/sarthak_negi/armada/third_party/airflow/docs> bash -c 'grep -v '"'"'^date:'"'"' build/jekyll/python_airflow_operator.md > /tmp/python_airflow_operator_generated.md'
docs-check: commands[2] /home/sarthak_negi/armada/third_party/airflow/docs> diff /tmp/python_airflow_operator_generated.md ../../../docs/python_airflow_operator.md
.pkg: _exit> python /home/sarthak_negi/armada38/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
  docs-check: OK (49.14=setup[48.71]+cmd[0.42,0.01,0.00] seconds)
  congratulations :) (49.18 seconds)

**Review by kannon92 — APPROVED** [review]

**Comment by sarthaksarthak9** [comment]
@Sharpz7 I think this pr is not merging

**Comment by kannon92** [comment]
Yea, I lost my approval rights so it only has 1/2 required approvers.

**Comment by sarthaksarthak9** [comment]
> Yea, I lost my approval rights so it only has 1/2 required approvers.

ohh

**Review by Mo-Fatah — APPROVED** [review]

**Pull request merged**